rpcs3/Emu/Cell/lv2/lv2.cpp

```diff
@@ -664,7 +658,7 @@ const std::array<ppu_function_t, 1024> s_ppu_syscall_table
 	null_func,//BIND_FUNC(sys_...)                          //696 (0x2B8)  ROOT
 	null_func,//BIND_FUNC(sys_...)                          //697 (0x2B9)  UNS
 	null_func,//BIND_FUNC(sys_...)                          //698 (0x2BA)  UNS
-	null_func,//BIND_FUNC(sys_bdemu_send_command)           //699 (0x2BB)
```

Comment:
Another one, definitely exists in retail kernel. I won't mention them all, just don't touch anything.